.evergreen/config.yml

```diff
@@ -322,10 +322,10 @@ axes:
       - id: java-master
         display_name: "Java (master)"
         variables:
-          DRIVER_DIRNAME: "java"
+          DRIVER_DIRNAME: "java/sync"
```

Comment:
Eventually we may want to test the reactive streams driver, so leaving room for that here.